python/GafferUI/MultiLineTextWidget.py

```diff
@@ -132,6 +133,14 @@ def getWrapMode( self ) :
 			QtGui.QTextOption.WrapAtWordBoundaryOrAnywhere : self.WrapMode.WordOrCharacter,
 		}[self._qtWidget().wordWrapMode()]
 
+	def setFixedLines( self, fixedLines ) :
```

Comment:
How about we call this `setFixedLineHeight()` so that it matches `TextWidget.setFixedCharacterWidth()`?. To be honest, I think `setFixedCharacterWidth()` sounds a bit clunky, but I'd rather they both sounded clunky in the same way - it makes it a bit easier to remember.